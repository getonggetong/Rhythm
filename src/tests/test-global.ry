def c;

init() {
	c = [A4,B4,C4];
	testGlobal();
}

testGlobal() {
	print(c);
}