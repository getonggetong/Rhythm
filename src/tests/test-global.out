[A4,B4,C4]